DiscordBot/Commands/EventCommands.cs

```diff
@@ -36,5 +43,10 @@ public async Task Event(CommandContext context)
                 await context.RespondAsync(response.Message.Content);
             }
         }
+
+        private bool IsValidResponse(string response, IEnumerable<string> validResponses)
```

Comment:
There's not really any reason to pass in the responses here, you can just use the class variable 🙂